cmd/minikube/cmd/start.go

```diff
@@ -88,6 +88,9 @@ type versionJSON struct {
 	Commit          string `json:"commit"`
 }
 
+// ErrPatchNotFound is when given volume was not found
```

Comment:
```suggestion
// ErrPatchNotFound is when given version was not found
```
I'm guessing that was meant to be version?

Reply:
Meant to be patch was not found. Should it be version instead?

Reply:
Well actually it would be when the patch isnt found when given the major.minor